*** tests/A-120-120-M-ok.out	2023-10-19 17:17:27.000000000 +0100
--- tests/A-120-120-M-ok.outhyp	2023-10-25 12:21:35.449824296 +0100
***************
*** 1,17 ****
  Menu Principal
  1 - Novo
  2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Novo
  2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
  0 - Sair
  Escolha uma opção: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
  1 - Novo
  2 - Abrir
--- 1,13 ----
  Menu Principal
  1 - Novo
  2 - Abrir
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu Principal
  1 - Novo
  2 - Abrir
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
  1 - Novo
  2 - Abrir
***************
*** 54,60 ****
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Guardar antes de fechar? Ficheiro a abrir: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
--- 50,57 ----
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Guardar antes de fechar? Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro saved30
! Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
***************
*** 70,76 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|'789=COALESCE(1;1:1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 67,73 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|5
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 80,86 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;4|1075=PRODUCT(2;1:2;2)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 77,83 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;4|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
